models/Trail.js

```diff
@@ -0,0 +1,42 @@
+const { Model, DataTypes } = require('sequelize');
+const sequelize = require('../config/connection');
+
+class Trail extends Model {}
+
+Trail.init(
+  {
+    //id, name, location, distance, elevation, difficulty
+    id: {
+        type: DataTypes.INTEGER,
+        allowNull: false,
+        primaryKey: true,
+        autoIncrement: true,
+    },
+    location: {
+        type: DataTypes.STRING,
+        allowNull: false,        
+    },
+    distance: {
```

Comment:
I think the necessary info for a trail its just its name and location, so suggest distance, elevation, and difficulty are permitted to accept null values.